src/lib/tabwidget/tabwidget.cpp

```diff
@@ -194,6 +194,15 @@ void TabWidget::loadSettings()
     m_newEmptyTabAfterActive = settings.value("newEmptyTabAfterActive", false).toBool();
     settings.endGroup();
 
+    settings.beginGroup("Web-Browser-Settings");
```

Comment:
I don't think this is wanted, it will overwrite any previously set zoom when applying preferences.